src/components/SignIn/Form.js

```diff
@@ -24,10 +25,12 @@ const SignInForm = () => {
   const formMethods = useForm({ validationSchema });
 
   const onSubmit = async (values) => {
+    setIsLoading(true);
     try {
       await dispatch(signIn(values));
       history.replace('/');
     } catch (error) {
+      setIsLoading(false);
```

Comment:
What do you think about putting this in the `finally` block as you did above? (Only to keep consistency)

Reply:
That would make it so we update the component after the `history.replace('/');` which would unmount the component if I'm not mistaken. Wouldn't it?

Reply:
Yep totally right! Didn't noticed 😅